cpp/ql/src/semmle/code/cpp/Class.qll

```diff
@@ -881,15 +934,16 @@ class ClassTemplateInstantiation extends Class {
   /**
    * Gets the class template from which this instantiation was instantiated.
    *
-   * Example: For `std::vector<float>`, returns `std::vector<T>`.
+   * For example: For `std::vector<float>`, the result is `std::vector<T>`.
```

Comment:
I wonder if it would be better using the same example (or, where necessary, variants thereof) throughout the class

Reply:
Done.